canonical/canonical.com
# Fix the department pages

## Done
Fix the department pages URLs

## QA

- Go to /careers and click the department links
- Check production
